Solutions/RubrikSecurityCloud/Playbooks/RubrikRansomwareDiscoveryAndVMRecovery/azuredeploy.json

```diff
@@ -2,4177 +2,4131 @@
+                                                                                                              "required": [
+                                                                                                                  "id",
+                                                                                                                  "status",
+                                                                                                                  "ipAddress",
+                                                                                                                  "__typename"
+                                                                                                              ],
+                                                                                                              "type": "object"
+                                                                                                          },
+                                                                                                          "type": "array"
+                                                                                                      }
+                                                                                                  },
+                                                                                                  "type": "object"
+                                                                                              },
+                                                                                              "connectivityLastUpdated": {
+                                                                                                  "type": "string"
+                                                                                              },
+                                                                                              "globalManagerConnectivityStatus": {
+                                                                                                  "properties": {
+                                                                                                      "__typename": {
+                                                                                                          "type": "string"
+                                                                                                      },
+                                                                                                      "urls": {
+                                                                                                          "items": {
+                                                                                                              "properties": {
+                                                                                         
```

Comment:
Hello @niralishah-crest, 
variables ( clusterId, migrationConfig, resourcePoolId & vNicBindings ) have empty strings. 
Please fill it or not needed remove it. It is causing arm ttk failure.

Reply:
@v-prasadboke I have removed above mentioned variables which have empty strings. But still getting same arm-ttk failure.